README.md

```diff
@@ -75,7 +28,6 @@ to hide your IP while using FreeTube.
 * Youtube Chapters
 * Most popular videos page based on the set Invidious instance
 * SponsorBlock 
-* Open videos from your browser directly into FreeTube (with extension)
 * Watch videos using an external player
```

Comment:
External player settings are hidden outside of electron. Technically, a user could open the link to a video file and copy it into their preferred player, but external players are not directly supported in Cordova atm.
```suggestion
```